.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js: node
+after_success: 
+    - npm run build
+deploy:
+    provider: s3
+    access_key_id: "AKIAJGKZPVSGGK6DK4JA"
```

Comment:
Yeah. They show how to do it on their site but it was throwing an error and I gave up with it last night.